drivers/wifi/nrf700x/osal/hw_if/hal/inc/fw/rpu_if.h

```diff
@@ -433,4 +433,19 @@ struct pta_ext_params {
 	unsigned int dec_time;
 } __NRF_WIFI_PKD;
 
+/**
+ * enum wifi_nrf_host_ref_rpu_ps_state - Indicates current RPU Power save state
+ * from host perspective
+ *
+ * @WIFI_NRF_HOST_REF_RPU_PS_STATE_SLEEP: From host perspective RPU is in Sleep state.
+ * @WIFI_NRF_HOST_REF_RPU_PS_STATE_AWAKE: From host perspective RPU is in Awake state.
+ * @WIFI_NRF_HOST_REF_RPU_PS_STATE_MAX: Invalid host perspective of RPU power save state.
+ *
+ * This enum lists the possible host perspective of RPU power save states.
+ */
+enum wifi_nrf_host_ref_rpu_ps_state {
```

Comment:
Why is this in `rpu_if.h` this is just local to host and we already have `enum RPU_PS_STATE` (this should be prefixed with WIFI_NRF).